tc/core/tc2halide.cc

```diff
@@ -724,11 +724,11 @@ void translateComprehension(
 
   if (added_implicit_initialization) {
     // Also reorder reduction initializations to the TC convention
-    vector<Var> lhs = func.args();
+    vector<Var> lhs2 = func.args();
```

Comment:
Not sure why this was called `lhs` to start with...  Can we call it `funcArgs` instead?